src/server/rest/v1/service/ChargingStationService.ts

```diff
@@ -164,7 +164,7 @@ export default class ChargingStationService {
         delete chargingStation.templateHashTechnical;
       // Manual config -> Auto Config || Auto Config with no Charge Point
       } else if ((chargingStation.manualConfiguration && !filteredRequest.manualConfiguration) ||
-        (!filteredRequest.manualConfiguration && !(chargingStation.chargePoints?.length > 0))) {
+        (!filteredRequest.manualConfiguration && !Utils.isEmptyArray(chargingStation.chargePoints))) {
```

Comment:
@jerome-benoit 
I think the '!' before the Utils method must be removed here. Otherwise the station will always restart when saving.